specs/phase0/p2p-interface.md

```diff
@@ -243,6 +244,10 @@ Each gossipsub [message](https://github.com/libp2p/go-libp2p-pubsub/blob/master/
 Clients MUST reject (fail validation) messages that are over this size limit.
 Likewise, clients MUST NOT emit or propagate messages larger than this limit.
 
+The optional `from` (1), `seqno` (3), `signature` (5) and `key` (6) protobuf fields are omitted from the message,
+since messages are identified by content, anonymous, and signed where necessary in the application layer.
+Starting from Gossipsub v1.1, clients MUST enforce this by applying the `StrictNoSign` signature policy. 
```

Comment:
Let's add a link to the libp2p/gossipsub spec when the other PR gets merged

```suggestion
Starting from Gossipsub v1.1, clients MUST enforce this by applying the `StrictNoSign` [signature policy](). 
```

Reply:
Updated with expected link, libp2p specs PR is near merge. (And mistakenly opened this PR from specs fork, my bad)